src/v3/src/components/Form/Layout.tsx

```diff
@@ -22,21 +23,23 @@ import {
   UISchemaLayoutType,
 } from '../../types';
 import { isDevelopmentEnvironment, isTestEnvironment } from '../../util';
+// eslint-disable-next-line import/no-cycle
+import Accordion from './Accordion';
```

Comment:
TBH, I think the existence of this "Layout" component is the source of the cycle. We should know at "tester" time which of the various layouts to use.

Reply:
Yea we used to have the Stepper in the Renderers.tsx file (testers) but it was refactored out, I cant remember if it was because of an issue or just when we switched from jsonforms. But yea we can figure it out with that ticket.